src/rnanorm/methods/between_sample.py

```diff
@@ -0,0 +1,417 @@
+"""Between sample normalizations."""
+import numpy as np
+from scipy.stats import rankdata, scoreatpercentile
+from sklearn.base import BaseEstimator, OneToOneFeatureMixin, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+
+from ..typing import Numeric1D, Numeric2D, Self
+from .utils import LibrarySize
+
+
+def remove_zero_genes(X: Numeric2D) -> Numeric2D:
+    """Remove genes with zero counts in all samples.
+
+    Also, convert to numpy.
+    """
+    X = np.asarray(X)
+    return X[:, np.sum(X, axis=0) > 0]
+
+
+def geometric_mean(x: Numeric1D) -> float:
```

Comment:
Scipy calls it `gmean`. We could use the same naming.

Reply:
Of course, let;'s use the scipy's function instead! Thank you!